modules/markup/sanitizer.go

```diff
@@ -120,6 +120,21 @@ func createDefaultPolicy() *bluemonday.Policy {
 	// Allow 'color' and 'background-color' properties for the style attribute on text elements.
 	policy.AllowStyles("color", "background-color").OnElements("span", "p")
 
+	// Allow classes for file preview links...
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^(lines-num|lines-code chroma)$")).OnElements("td")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^code-inner$")).OnElements("code")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^file-preview-box$")).OnElements("div")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^ui table$")).OnElements("div")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^header$")).OnElements("div")
+	policy.AllowAttrs("data-line-number").OnElements("span")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^text small grey$")).OnElements("span")
+	policy.AllowAttrs("rel").OnElements("td")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^file-preview*")).OnElements("table")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^lines-escape$")).OnElements("td")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^toggle-escape-button btn interact-bg$")).OnElements("a")
+	policy.AllowAttrs("class").Matching(regexp.MustCompile("^ambiguous-code-point$")).OnElements("span")
+	policy.AllowAttrs("data-tooltip-content").OnElements("span")
+
```

Comment:
This seems too brittle. I wonder if we can somehow bypass the sanitizer completely for node and all its children, ideally still forbidding all these attributes from user content.

Reply:
Yeah I thought so too. It's also why I made the button added in the PR an `<a>` to not allow `<button>`'s from user content.

The problem is that all postprocessing is applied before the sanitizer is called, and since all the `*html.Node` is written to a writer and thus looks like normal / user-created html, theres no simple way to distinguish generated html and user-provided one.

It might be possible to do two postprocessing passes, one before and one after sanitizing, or to move post processing after the santizing step since all post processing is declared inside go so there should no risk here, atleast if each postprocessing step takes some care to escape / sanitize any outside input themself.

Reply:
I see. I guess it is acceptable for now with the current method, but this is definitely something we need to do eventually.